README.rst

```diff
@@ -304,15 +288,15 @@ The C++ code is tested with
 Lightning-Kokkos installation
 *****************************
 
-On linux systems, ``lightning.kokkos`` with the OpenMP backend can be installed by providing the optional ``[kokkos]`` tag:
-
-.. code-block:: console
-
-    $ pip install pennylane-lightning[kokkos]
+On most Linux systems, Lightning-Kokkos can be installed via Spack or Docker by following our installation instructions at `pennylane.ai/install <https://pennylane.ai/install/#high-performance-computing-and-gpus>`_.
```

Comment:
with lightning.kokkos, I believe there may be more cases than qubit/gpu where you need to install from source (e.g., if you want to use GPUs)

Reply:
Thanks! Does this sentence need to be modified 🤔?